### ISSUE TITLE
Dates with 0 second components formatted incorrectly

### ISSUE THREAD

**Issue by lkitching** [yetanalytics/flint#34]
Dates with 0 second components are formatted incorrectly within UPDATE queries. For example, the following query:

```clojure
{:insert-data [["<http://s>" "<http://p>" (OffsetDateTime/parse "2023-07-27T16:32:00Z")]]}
```

is formatted as the following

```
INSERT DATA { <http://s> <http://p> \"2023-07-27T16:32Z\"^^<http://www.w3.org/2001/XMLSchema#dateTime> . }
```

The [xsd:dateTime specification](https://www.w3.org/TR/xmlschema11-2/#dateTime) states that all components except the timezone offset are required, so the above is not a valid representation of the input date.

The current [implementation](https://github.com/yetanalytics/flint/blob/93cf11e7a0abd58982565344bca52b2f1d31c1dd/src/main/com/yetanalytics/flint/axiom/impl.cljc#L335) appears to use `.toString` to format datetime values, but the [OffsetDateTime implementation](https://docs.oracle.com/javase/8/docs/api/java/time/OffsetDateTime.html#toString--) omits the second component if zero.

**Comment by kelvinqian00** [comment]
@lkitching Apologies for the very late response, but we now have a PR open to address this formatting issue.

**Comment by kelvinqian00** [comment]
@lkitching This bug is now fixed in v0.3.0, in #38
